tests/microbenchmarks.py

```diff
@@ -242,11 +274,14 @@ class Z: pass
         e = list([1])
         f = True
         g = 3.0
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_method(test_case):
     print("Running, test_method")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```

Comment:
At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils